lib/DevServerManager.ts

```diff
@@ -133,11 +132,7 @@ class DevServerManager {
         }
       });
 
-      const hasActiveServers = await portManagerHasActiveServers();
-
-      if (!hasActiveServers) {
-        await stopPortManagerServer();
-      }
```

Comment:
This check was seeing if any servers had registered with the `portManager` and hot if it was running.  The `stopPortManagerServer` method already checks if the server is running before stopping it.